### DISCUSSION TITLE
'error:03000086:digital envelope routines::initialization error' After NodeJs Upgrade

### DISCUSSION THREAD

**Discussion by mecjos** [quasarframework/quasar#12050] [General - Components / Directives / etc] [answered]
Hi, I upgraded nodejs to latest version because latest quasar app didn't support my node. After upgrade I got following error :

`'error:03000086:digital envelope routines::initialization error'`

After a research I found following solution : 

`export NODE_OPTIONS=--openssl-legacy-provider`

But I don't know how I will put this into quasar build options. Or How can I fix this problem ?

Thank you

**Comment by MilosPaunovic** [comment]
Which exact `Node` version have you installed?

**Comment by hawkeye64** [comment, marked as answer]
@mecjos Whenever you upgrade Node, you have to delete your node_modules folder and your lock file, then re-install.
Also, in case it is another issue, do not use odd-numbered versions of Node as these are considered experimental and sometimes have side effects.

Personally, I am using v16.+ LTS with no issues.

**Reply by mecjos** [reply]
Thank you.. I downgraded to v16 and solved.

**Reply by Deepak-Gautam** [reply]
I am having same issue with v18.12.1LTS
